### ISSUE TITLE
Could not retrieve firmware version and module ID

### ISSUE THREAD

**Issue by manjunathgudisi** [tessel/t2-cli#1355]
Getting foolowing error when I run the app on Tessel2.

`$ t2 run smarthome.js`

```
INFO Looking for your Tessel...
INFO Connected to Tessel-02A3B88B77F4.
INFO Building project.
INFO Writing project to RAM on Tessel-02A3B88B77F4 (1426.432 kB)...
INFO Deployed.
INFO Running smarthome.js...
(node:1587) DeprecationWarning: pin.rawWrite is deprecated. Use .high() or .low()
Could not retrieve firmware version and module ID. Attempting update.
Error: Unable to communicate with module. Are you sure it's plugged in?
    at Error (native)
    at r (/tmp/remote-script/node_modules/attiny-common/lib/index.js:1:615)
    at /tmp/remote-script/node_modules/attiny-common/lib/index.js:1:897
    at /tmp/remote-script/node_modules/attiny-common/lib/index.js:1:2318
    at /tmp/remote-script/node_modules/attiny-common/lib/attiny-isp.js:1:1262
    at /tmp/remote-script/node_modules/attiny-common/lib/attiny-isp.js:1:157
    at /tmp/remote-script/node_modules/avr-isp/index.js:1:773
    at Port.<anonymous> (/tmp/remote-script/node_modules/avr-isp/index.js:1:3831)
    at Socket.Port.sock.on (/usr/lib/node/tessel-export.js:445:33)
    at emitNone (events.js:86:13)
```

Attached screenshot

![screen shot 2017-10-22 at 7 43 23 pm](https://user-images.githubusercontent.com/6059076/31862810-d76d21b0-b761-11e7-8da6-6dcba511cacb.jpg)

**Comment by HipsterBrown** [comment]
When looking into the attiny-common source, it appears the rawWrite method was deprecated in t2-firmware and not updated in attiny-common to use the low() method. 

What module are you using?

**Comment by manjunathgudisi** [comment]
Im using Ambient and Climate modules.

**Comment by HipsterBrown** [comment]
Can you link to some example code of how you're using the `attiny-common` package to communicate with those modules?

**Comment by manjunathgudisi** [comment]
Here is my code. Thanks for your help in advance.
[smarthome.zip](https://github.com/tessel/t2-cli/files/1425977/smarthome.zip)

**Comment by manjunathgudisi** [comment]
HI, does anyone help me here. Here the situation is critical now. Thanks, MG

**Comment by manjunathgudisi** [comment]
I have fixed the issue by removing `attiny-common` from package.json and updated `t2-cli` with latest version.

**Comment by HipsterBrown** [comment]
@manjunathgudisi The `attiny-common` package has been patched and released as version 0.1.1.
